src/quri.lisp

```diff
@@ -106,6 +106,9 @@
            :uri-malformed-urlencoded-string))
 (in-package :quri)
 
+(defparameter *scheme-list*
+  '("http" "https" "ldap" "ldaps" "ftp" "file" "urn"))
```

Comment:
This hard-coding is brittle, in particular it prevents one from extending the library.
Turn it to a function instead:

```lisp
(mapcar #'class-name (mopu:subclasses 'quri:uri))
```

But `quri` does not depend on `mopu`.  So instead of including another dep, you could also add a metaclass` on `quri:uri` to keep track of the scheme.  Hmmm... The mopu way seems easier.